packages/story-editor/package.json

```diff
@@ -38,6 +38,7 @@
         "flagged": "^2.0.1",
         "history": "^5.0.1",
         "html-to-image": "^1.7.0",
+        "lodash.groupby": "^4.6.0",
```

Comment:
@swissspidy @spacedmonkey What do y'all think about me adding this in? Should we write a util instead?

here's the sizing
![image](https://user-images.githubusercontent.com/22185279/132763819-33bc28ed-5fdf-4473-9099-6e03be3b74c1.png)

Reply:
So far we've been pretty strict/consistent with not using any lodash utils and I'd prefer to keep it this way.

Reply:
I'll add a custom util tomorrow. Thanks!

Reply:
Ended up adding one right now -> not using lodash anymore 🙌

Reply:
I know that @barklund was really against using lodash.